Directory.Build.targets

```diff
@@ -6,13 +6,20 @@
                                 '$(IsTestUtilityProject)' != 'true'">true</IsSourceProject>
   </PropertyGroup>
 
+  <ItemDefinitionGroup>
```

Comment:
Please add a comment above the `ItemDefinitionGroup` to indicate that this is used by the below target.